packages/frontend/amp/components/BodyLiveblog.tsx

```diff
@@ -155,17 +160,28 @@ export const Body: React.FC<{
             {!isFirstPage && (
                 <Pagination guardianURL={url} pagination={data.pagination} />
             )}
-            <Blocks
-                pillar={pillar}
-                blocks={data.blocks}
-                // stuff for ads
-                edition={data.editionId}
-                section={data.sectionName}
-                contentType={data.contentType}
-                switches={config.switches}
-                commercialProperties={data.commercialProperties}
-                url={url}
-            />
+            <amp-live-list
+                id="live-blog-entries-7ea0dbef"
+                data-poll-interval="15000"
```

Comment:
This is the default 🤷‍♂

Reply:
Yes. Recommended default in the documentation.

Reply:
Yeah, so you can remove it?

Reply:
Um.......... Yeah.... The docs say

> If no `data-poll-interval` is provided it will default to the 15000 ms minimum.

So I guess I can do that :)